components/workspacekit/cmd/rings.go

```diff
@@ -669,32 +669,32 @@ func receiveSeccmpFd(conn *net.UnixConn) (libseccomp.ScmpFd, error) {
 
 	err := conn.SetDeadline(time.Now().Add(5 * time.Second))
 	if err != nil {
-		return 0, err
+		return 0, xerrors.Errorf("cannot setdeadline: %v", err)
 	}
 
 	f, err := conn.File()
 	if err != nil {
-		return 0, err
+		return 0, xerrors.Errorf("cannot open socket: %v", err)
 	}
 	defer f.Close()
 	connfd := int(f.Fd())
 
 	_, _, _, _, err = unix.Recvmsg(connfd, nil, buf, 0)
 	if err != nil {
-		return 0, err
+		return 0, xerrors.Errorf("cannot recvmsg from fd '%d': %v", connfd, err)
 	}
 
 	msgs, err := unix.ParseSocketControlMessage(buf)
 	if err != nil {
-		return 0, err
+		return 0, xerrors.Errorf("cannot parse socket control message: %v", err)
```

Comment:
does it make sense to add more info here and other messages in the later if blocks? e.g. `connfd`

Reply:
I don't think so - the message is essentially a binary blob